examples/tty_conv.c

```diff
@@ -0,0 +1,165 @@
+/* PlanC (hubenchang0515@outlook.com) -- an example application
+ * that implements a custom conversation */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <termio.h>
+#include <security/pam_appl.h>
+
+/***************************************
+ * @brief echo off/on
+ * @param[in] fd file descriptor
+ * @param[in] off 1 - echo off，0 - echo on
+ ***************************************/
+static void echoOff(int fd, int off)
+{
+    struct termio tty;
+    (void) ioctl(fd, TCGETA, &tty);
+    if (off)
+    {
+        tty.c_lflag &= ~(ECHO | ECHOE | ECHOK | ECHONL);
+        (void) ioctl(fd, TCSETAF, &tty);
+    }
+    else
+    {
+        tty.c_lflag |= (ECHO | ECHOE | ECHOK | ECHONL);
+        (void) ioctl(fd, TCSETAW, &tty);
+    }
```

Comment:
Why would you want to perform an ioctl TCSETAW operation on a descriptor if ioctl TCGETA failed?
A good example code shouldn't ignore the return value of ioctl calls, should it?